### DISCUSSION TITLE
Slide before ToC in revealjs presentation

### DISCUSSION THREAD

**Discussion by OlegHahm** [quarto-dev/quarto-cli#7606] [Feature Requests]
### Description

Apologies if this question has been discussed already but I couldn't find an answer: is it possible to insert a slide before the auto generated ToC slide?

**Comment by cderv** [comment]
No I don't think this is possible without tweaking the revealjs Template. 

See about Pandoc template support in Quarto with PDF article example: https://quarto.org/docs/journals/templates.html

It works the same with revealjs, where you could provide a custom template. 

In Quarto template leave here (https://github.com/quarto-dev/quarto-cli/tree/main/src/resources/formats/revealjs/pandoc) and you could only modify the TOC partials (https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/formats/revealjs/pandoc/toc-slide.html) to add some slide content (as HTML though, or adding YAML variable to set in your document). 

I'll move this to feature request in case we want to consider simplifying this one day

**Reply by mcanouil** [reply]
Lua filter could also be used.

In some aspects related to this, there is the Quarto extension <https://github.com/andrie/reveal-auto-agenda>.
